src/astra_driver.cpp

```diff
@@ -42,6 +42,8 @@
 #include "astra_camera/image_encodings.h"
 #include "astra_camera/distortion_models.h"
 
+#include "sensor_msgs/msg/camera_info.hpp"
```

Comment:
Yeah, it does get pulled in by astra_driver.h.  However, I tend to list all of the headers needed for each particular file anyway.  For now, I'll include as a system header, you can let me know if you still want me to remove it.